Dockerfile

```diff
@@ -70,6 +70,7 @@ RUN mix deps.clean --all
 RUN mix deps.get
 
 RUN mix compile
+RUN mix sentry.recompile
```

Comment:
why is this needed?

Reply:
@InoMurko https://github.com/getsentry/sentry-elixir#reporting-exceptions-with-source-code